include/rocksdb/table_properties.h

```diff
@@ -136,6 +136,10 @@ class TablePropertiesCollectorFactory {
 
   // The name of the properties collector can be used for debugging purpose.
   virtual const char* Name() const = 0;
+
+  // Can be overridden by sub-classes to return configuration info that will
+  // be logged to the info log when the DB is opened
+  virtual std::string ToString() const { return ""; }
```

Comment:
Since this does not include the name, I'm wondering if something like `DebugInfo`, `Parameters` or `Configuration` would be a better name. Alternatively, we could keep `ToString` and make it include the name as well, in which case by default it could return what `Name` returns (as opposed to an empty string).

Reply:
I like the latter idea.